kinto/core/resource/schema.py

```diff
@@ -6,6 +6,10 @@
                                FieldList, TimeStamp, URL)
 from kinto.core.utils import native_value
 
+POSTGRESQL_MAX_INTEGER_VALUE = 2**64 / 2
```

Comment:
The tests are probably failing because Python3 implement default float casting on division. Try `2**64 // 2` or `2**32`